doc/contributing/coding-conventions.chapter.md

```diff
@@ -171,7 +285,7 @@
 
 - Arguments should be listed in the order they are used, with the exception of `lib`, which always goes first.
 
-- The top-level `lib` must be used in the master and 21.05 branch over its alias `stdenv.lib` as it now causes evaluation errors when aliases are disabled which is the case for ofborg.
+- The top-level `lib` must be used in the master and 21.05 branches over its alias `stdenv.lib` as it now causes evaluation errors when aliases are disabled (which is the case for the `ofborg` GitHub bot).
```

Comment:
```suggestion
- The top-level `lib` must be used in the master and 21.05 release branche over its alias `stdenv.lib` as it now causes evaluation errors when aliases are disabled (which is the case for the `ofborg` bot).
```

Reply:
Do you mean this to say "master and 21.05 release branches" or "master branch and 21.05 release branch"? If it's the former, I don't think master is a RB, and if it's the latter, I'm not sure it's necessary.